tests/functional/object/test_object_browser_client_provider_ui.py

```
@@ -500,3 +512,359 @@ def test_object_browser_list_objects_with_folders(self, project_factory):
+            presigned_url = s3_client.generate_presigned_url(
+                "get_object",
+                Params={"Bucket": bucket_name, "Key": test_object_key},
+                ExpiresIn=3600,
+            )
+            logger.info("Generated presigned URL: %s", presigned_url)
+
+            # Step 3: Validate URL is accessible
+            logger.test_step("Step 3: Validating presigned URL is accessible")
+            import requests
+
+            response = requests.get(presigned_url, verify=False)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)